### PR TITLE
Order show us28

### PR DESCRIPTION
**Pull request by GDemps**
[Reference story or milestone being closed]

**Contributors**
- Driver(s): Grant
- Navigator(s): Leah
  
**What was changed:**
Added profile order show page with info on orders
Added total_quantity and status_string class methods to order.rb with tests

**Any tech notes (if applicable)**

### CODE REVIEW

File: spec/features/users/profile/orders/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        within ("#order-#{@order_1.id}") do
+          expect(page).to have_link("Order: #{@order_1.id}")
+          expect(page).to have_content("Order date: #{@order_1.created_at}")
+          expect(page).to have_content("Last update: #{@order_1.updated_at}")
+          expect(page).to have_content("Status: #{@order_1.status_string}")
+          expect(page).to have_content("Quantity: #{@order_1.total_quantity}")
+          expect(page).to have_content("Total price: #{@order_1.grandtotal}")
+        end
+      end
```
Comment (leahriffell) [inline spec/features/users/profile/orders/index_spec.rb:26]: @GDemps The first test looks great! Could you please add a second test to capture the other part of the user story ("can link to order show page")?. Should just be a simple one of visiting the link and expecting current path to equal that order show page.

**Review by leahriffell — CHANGES_REQUESTED** [review]
Thanks, Grant! I requested a change for second test. 

It looks like there's lots of changes in here not related to the current user story. Are those work that you did or are they from a different PR?

File: spec/features/users/profile/orders/index_spec.rb (hunk 1 above)
Comment (GDemps) [reply spec/features/users/profile/orders/index_spec.rb:28]: Added path test for order show page.

Comment (leahriffell) [reply spec/features/users/profile/orders/index_spec.rb:28]: Thank you, @GDemps!

**Review by leahriffell — APPROVED** [review]

**Pull request merged**